s3/pkg/datastore/yig/meta/db/tidb/cluster.go

```diff
@@ -0,0 +1,16 @@
+package tidb
+
+import (
+	"github.com/opensds/multi-cloud/s3/pkg/datastore/yig/meta/types"
+)
+
+//cluster
```

Comment:
is this YIG CEPH specific ? cluster == backend ? in multicloud case ?

Reply:
@rhsakarpos yig backend will manage more than one ceph cluster, so yig backend is a top management layer. So, backend will contain more than one ceph cluster.